rolling/distribution.yaml

```diff
@@ -4117,6 +4117,25 @@ repositories:
       url: https://github.com/ros2/rcutils.git
       version: rolling
     status: maintained
+  realsense2_camera:
+    doc:
+      type: git
+      url: https://github.com/IntelRealSense/realsense-ros.git
+      version: ros2-development
+    release:
+      packages:
+      - realsense2_camera
+      - realsense2_camera_msgs
+      - realsense2_description
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/IntelRealSense/realsense-ros-release.git
```

Comment:
The same comment as https://github.com/ros/rosdistro/pull/37808#discussion_r1247989248 applies here.

Reply:
Opened PR, waiting for merge:
https://github.com/ros2-gbp/ros2-gbp-github-org/pull/297